src/main/java/helpers/DataHelper.java

```diff
@@ -0,0 +1,51 @@
+package helpers;
+
+import com.github.javafaker.Faker;
+
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.util.Random;
+
+public class DataHelper {
+    private static Faker faker = new Faker();
+    private static Random random = new Random();
+
+    public static String getRandomEmail() {
+        return faker.internet().emailAddress();
+    }
+
+    public static String getRandomText(int number) {
+        String result = "";
+        for(int i = 1;i <= number; i++){
+            result += "?";
+        }
+        return faker.letterify(result);
+    }
+
+    public static String getRandomText() {
+        return getRandomText(8);
```

Comment:
don't use magic number. should declare a variable here. like DEFAULT_LENGHT = 8;

Reply:
Thank you. I fixed it.